.env.tpl

```
@@ -1,5 +1,6 @@
 # set these to your upload API service URL and the DID your service is using as its service DID
 NEXT_PUBLIC_W3UP_SERVICE_URL=https://staging.up.web3.storage
+NEXT_PUBLIC_W3UP_RECEIPTS_URL=https://staging.up.web3.storage/receipt
```

Needs to be specified with a trailing slash as it is joined with the task CID like this https://github.com/storacha/w3up/blob/b7bc90e6a33b8f33dd7356794c55a77b364799ee/packages/upload-client/src/receipts.js#L89-L92

Consistent with https://github.com/storacha/w3up/blob/b7bc90e6a33b8f33dd7356794c55a77b364799ee/packages/upload-client/src/service.js#L7

```suggestion
NEXT_PUBLIC_W3UP_RECEIPTS_URL=https://staging.up.web3.storage/receipt/
```